clap_complete/src/dynamic/shells/mod.rs

```diff
@@ -18,6 +18,69 @@ use crate::dynamic::Completer as _;
 /// A subcommand definition to `flatten` into your CLI
 ///
 /// This provides a one-stop solution for integrating completions into your CLI
+///
+/// # Examples
```

Comment:
btw this is a good example of something that could be split into its own PR.  While this PR is blocked on confirming if any zsh functionality is missing (for whether it closes the zsh issue), the docs are nearly ready to merge.

Reply:
Thanks! I'd like to split it into its own PR.

Reply:
I split the docs commit into its own PR https://github.com/clap-rs/clap/pull/5586